src/main/java/net/schmizz/sshj/sftp/Response.java

```diff
@@ -29,7 +29,7 @@
         FAILURE(4),
         BAD_MESSAGE(5),
         NO_CONNECTION(6),
-        CONNECITON_LOST(7),
+        CONNECTION_LOST(7),
```

Comment:
This is possibly an API break

Reply:
It might indeed break the API, though the Response packet is typically not exposed. But let's remove it from the PR